File: server/order-service/src/main/java/com/sudo248/invoiceservice/controller/dto/AddressDto.java
package com.sudo248.invoiceservice.controller.dto;

import com.sudo248.invoiceservice.repository.entity.Location;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class AddressDto {
    private String addressId;

    private int provinceID;

    private int districtID;

    private String wardCode;

    private String provinceName;

    private String districtName;

    private String wardName;

    private String address;

    private Location location;

    private String fullAddress;
}

File: client/mobile/Sudoo/app/src/main/java/com/sudo248/sudoo/data/dto/user/AddressDto.kt
package com.sudo248.sudoo.data.dto.user

import com.sudo248.sudoo.domain.entity.user.Location

data class AddressDto(
    val addressId: String,
    val provinceID: Int,
    val districtID: Int,
    val wardCode: String,
    val provinceName: String,
    val districtName: String,
    val wardName: String,
    val address: String,
    val location: Location? = null,
    val fullAddress: String,
)